requirements.txt

```diff
@@ -26,7 +26,7 @@ Markdown==2.6.11
 MarkupSafe==1.1.1
 mywsgi==1.0.3
 more-itertools==4.2.0
-mypy>=0.800
+mypy>=0.812
```

Comment:
Just so you know, there is mypy 0.900, and when Sentry upgraded to that version is broke a bunch of the typing.